Resources/views/CRUD/edit_orm_many_to_many.html.twig

```diff
@@ -9,7 +9,7 @@ file that was distributed with this source code.
 
 #}
 {% if sonata_admin.field_description.hasassociationadmin %}
-    <div id="field_container_{{ id }}" class="field-container">
+    <div id="field_container_{{ id }}" class="field-container {{ form.vars["attr"]["class"] is defined ? form.vars["attr"]["class"] }}">
```

Comment:
> There are longer lines in the same file

Yes, and that's sad. We're trying to fix that gradually.

For php, there is a standard : 

> There MUST NOT be a hard limit on line length; the soft limit MUST be 120 characters; lines SHOULD be 80 characters or less.

So as you see, it's not a hard rule, just a recommendation, because there might be some cases where this is impossible to fix (I don't know which, but…)

> Will this PR get blocked because the line looks long?

I hope so. We should stop with that crap. We already have to scroll vertically, if we have to also scroll horizontally, what's next? Scrolling in the third dimension ?